README.md

```diff
@@ -151,16 +151,13 @@ app.use(async (ctx, next) => {
 
 For more details please refer to: [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#server-side-rendering)
 
-## Building
+## Lint and test
 
 ```bash
 npm install
-npm install gulp -g
-gulp build
+npm run ci
```

Comment:
this script should only be run by a CI environment. I have some plans to do some funky stuff in Travis that shouldn't be performed on a fork/local machine. `npm run lint` and `npm run test` should both be listed here.